.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
```

Comment:
what is this code doing here?
 just for my information

Reply:
my vscode automatically adds it. i will add gitignore on the next push